Web Development/README.md

```diff
@@ -348,6 +348,11 @@ to everyone - everywhere!</td>
         <td>Hitesh Choudhari is a YouTube channel that offers various courses on programming languages. It is one of the best channel to learn web development at free of cost.
         </td>
       </tr>
+      <tr>
+        <td><a href="">https://www.youtube.com/watch?v=bMknfKXIFA8</a>FreeCodeCamp</td>
+        <td>React course for beginners. It simplifies and dymystifies react to any beginner who is willing and eager to learn.
+        </td>
+      </tr>
```

Comment:
Hello @RitaDee It looks good to me but the problem is that this yt link you added is suitable for `Turorial or Courses` section. Now please can you move it to that section and then I will merged once you're finished. Thank you!

Reply:
Done. Thank you